roles/config-nagios-target/tasks/nrpe.yml

```diff
@@ -2,29 +2,29 @@
 
 - name: Include Docker related configuration (if enabled)
   when: '"docker" in nagios_services'
-  include: nrpe_docker.yml
+  include_tasks: nrpe_docker.yml
 
 - name: Include DNS related configuration (if enabled)
   when: '"dns" in nagios_services'
-  include: nrpe_dns.yml
+  include_tasks: nrpe_dns.yml
 
 - name: Include NFS related configuration (if enabled)
   when: '"nfs" in nagios_services'
-  include: nrpe_nfs.yml
+  include_tasks: nrpe_nfs.yml
 
 - name: Include Memory related configuration
-  include: nrpe_mem.yml
+  include_tasks: nrpe_mem.yml
```

Comment:
As this isn't dynamically included, it should use `import_tasks`.

Same applies below for the non-dynamic ones.